docs/guides/jupyter.rst

```diff
@@ -0,0 +1,293 @@
+How to use Jupyter notebooks in Sphinx
+======================================
+
+By default, `Sphinx only supports reStructuredText source files for
+documentation <https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-source_suffix>`_.
+Luckily, there are a few extensions to also allow Jupyter notebooks as source files.
+
```

Comment:
This feels like a pretty abrupt introduction. We should cover a bit more about what this tutorial is about and why we're writing it. 

```suggestion
Jupyter notebooks are a common format to represent calculations that have been done on a set of data.
Embedding them in your Sphinx documentation allows an integration between data calculations and documentation,
which can provide a great experience for examples and other pages in your documentation. 
There are a few extensions that allow integrating these two tools,
and this document will explain how to achieve some of the most commonly requested features. 

```

Reply:
Good point. I took parts of this and rewrote them slightly.